addons/animated-thumb/addon.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Thumbnails Setter",
+  "description": "This addon will allow you to set the thubnail of your project to any photo (including gif images)",
```

Comment:
Maybe "any image", because it doesn't need to be a photo?

Reply:
I'll do that...